app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java

```diff
@@ -180,8 +180,14 @@ private void bindViewHolder(@NonNull RecyclerView.ViewHolder viewHolder, int pos
                 case VIEW_TYPE_STATUS: {
                     StatusViewHolder holder = (StatusViewHolder) viewHolder;
                     StatusViewData.Concrete status = concreteNotificaton.getStatusViewData();
-                    holder.setupWithStatus(status,
-                            statusListener, statusDisplayOptions, payloadForHolder);
+                    if (status == null) {
```

Comment:
```suggestion
                    // this can happen rarely on the server, seems to be a race condition
                    if (status == null) {
```